src/main/java/io/gravitee/policy/basicauth/BasicAuthenticationPolicy.java

```diff
@@ -44,6 +45,7 @@ public class BasicAuthenticationPolicy {
 
     private static final String BASIC_AUTHENTICATION_VALUE = "BASIC ";
     private static final String DEFAULT_REALM_NAME = "gravitee.io";
+    private static final String BASIC_AUTHENTICATION_UNAUTHORIZED = "BASIC_AUTHENTICATION_UNAUTHORIZED";
```

Comment:
To keep consistency between our policies, we could fit the behavior of the API Key policy, which returns 2 distinct error keys : [see doc](https://docs.gravitee.io/apim/3.x/apim_publisherguide_response_templates.html#policy_error_keys_you_can_override)

Maybe we could keep the same pattern, and have something like :
- BASIC_AUTHENTICATION_MISSING
- BASIC_AUTHENTICATION_INVALID

WDYT ?